.github/workflows/sync-labels.yml

```diff
@@ -1,16 +1,13 @@
-name: Sync labels from UX Guide repo
+name: Sync labels
 
-on:
- schedule:
-   # daily at 3am UTC so as to probably not disrupt anyone
-   - cron: '0 3 * * *'
+on: [label]
 
 jobs:
   labels:
-    name: synchronize labels from UX Guide repo
+    name: synchronize labels between repos
     runs-on: ubuntu-latest
     steps:
-      - name: sync labels from UX Guide repo
+      - name: sync labels between repos
```

Comment:
This whole edit is just about making it the same as the one in the UX guide, except for line 13 which has to be different between the two.